src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -441,6 +441,11 @@ public void canRunNativeImage() throws IOException, InterruptedException {
     public void canRunLLVM(Triplet triplet) throws IOException, InterruptedException {
         if (!new Triplet(Constants.Profile.IOS).equals(triplet) &&
                 !new Triplet(Constants.Profile.ANDROID).equals(triplet)) {
+            // host doesn't use LLVM
+            return;
+        }
+        if (new Triplet(Constants.Profile.ANDROID).equals(triplet) && getBackend().equals(Constants.BACKEND_LIR)) {
```

Comment:
Better to reverse that test (if backend is not LLVM, return).
There might be different non-LLVM backends, or at least different names for this backend.

Reply:
Done